pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSinkConfig.java

```diff
@@ -60,7 +60,7 @@
             "The batch size that Kafka producer will attempt to batch records together before sending them to brokers.")
     private long batchSize = 16384L;
     @FieldDoc(
-        defaultValue = "16384",
+        defaultValue = "1048576",
```

Comment:
long type value is 1048576L, so defaultValue is 1048576l or 1048576L.

Reply:
I just [PR](https://github.com/apache/pulsar/blob/d0f8aa6c7d06ad3ad22b373c390fd7b1db61d434/pulsar-io/nifi/src/main/java/org/apache/pulsar/io/nifi/NiFiConfig.java#L71) . if value int value, I also fix this.